### PR TITLE
Feature/job site api

### PR DESCRIPTION
**Pull request by pkvarnwal**
- [x] I have reviewed the pull request myself. 
- [x] I confirm that code style is correct and follows CrownStack code style guide.
- [x] I have checked the code after running on Simulator/emulator/Device (which ever applicable).
- [x] I am convinced that I have not added new bugs. 
- [x] The UI behaviour is correct.

### CODE REVIEW

**Comment by ashutoshsingh33** [comment]
@aashishdhawan  Please review.

File: app/src/main/java/com/eqdepot/adapters/MyJobSiteAdapter.java [hunk 1]
```diff
@@ -88,27 +92,27 @@ public View getView(int position, View convertView, ViewGroup parent) {
...
         private void bindData(int position) {
+            this.position = position;
             jobsiteData = mJobsiteDatas.get(position);
             if (position % 2 == 1) {
                 linearLayoutRow.setBackgroundColor(ContextCompat.getColor(mActivity, R.color.eqdepot_light_black));
             }
             if (!mSource.equals(POPUP)) {
                 linearLayoutButton.setVisibility(View.VISIBLE);
-                if (jobsiteData.getIsDefault().equals("true")) {
+                if (jobsiteData.getCStatus().equals(1)) {
```
Comment (aashishdhawan) [inline app/src/main/java/com/eqdepot/adapters/MyJobSiteAdapter.java]: wrap this function

File: app/src/main/java/com/eqdepot/adapters/MyJobSiteAdapter.java [hunk 2]
```diff
@@ -120,15 +124,17 @@ private void bindData(int position) {
             }
             textViewPosition.setText(String.valueOf(position + 1));
             textViewName.setText(jobsiteData.getName());
-            textViewAddress.setText(jobsiteData.getAddressStreet() + ", " + jobsiteData.getAddressLandmark() + ", " + jobsiteData.getAddressState()
-                    + ", " + jobsiteData.getAddressCountry() + ", " + jobsiteData.getAddressPincode());
+            String address = jobsiteData.getAddressStreet() + ", " + jobsiteData.getAddressLandmark() + ", " + jobsiteData.getAddressState()
```
Comment (aashishdhawan) [inline app/src/main/java/com/eqdepot/adapters/MyJobSiteAdapter.java]: Please wrap this in JobSite object itself.

File: app/src/main/java/com/eqdepot/adapters/MyJobSiteAdapter.java [hunk 3]
```diff
@@ -138,12 +144,37 @@ public void onDismiss(boolean isDismiss) {
...
+        String session = AppUtils.getInstance().getSessionId(mActivity).getSessionId();
+        UpdateJobsiteRequest updateJobsiteRequest = new UpdateJobsiteRequest();
+        updateJobsiteRequest.setId(String.valueOf(jobsiteData.getId()));
+
+        ProgressHelper.start(mActivity, mActivity.getString(R.string.please_wait));
+        DataManager.getInstance().updateJobSiteStatus(mActivity, accessToken, session, updateJobsiteRequest, new DataManager.DataManagerListener() {
+            @Override
+            public void onSuccess(Object response) {
+                ProgressHelper.stop();
+                if (jobsiteData.getCStatus() == 0) jobsiteData.setCStatus(1);
```
Comment (aashishdhawan) [inline app/src/main/java/com/eqdepot/adapters/MyJobSiteAdapter.java]: Please wrap this in a function

**Pull request merged**